File: CoreJava/vehical-interface-demo-app/src/com/techlab/vehicals/Truck.java
package com.techlab.vehicals;

import com.techlab.imoveable.IMoveable;

public class Truck implements IMoveable {

	@Override
	public void start() {
		System.out.println("Truck Starts....");
	}

	@Override
	public void stop() {
		System.out.println("Truck Stops....");
	}

}

File: C#Basics/Interface-App/Vehical-App/Truck.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Vehical_App
{
    class Truck : IMoveable
    {
		void IMoveable.start()
		{
			Console.WriteLine("Truck Starts....");
		}

		
		void IMoveable.stop()
		{
			Console.WriteLine("Truck Stops....");
		}
	}
}
